### PR TITLE
feat(argocd-notifications): Add example for defaultTriggers

### PR DESCRIPTION
**Pull request by shortsn**
👋 Hi there,

i have added the possibility to define the [defaultTriggers](https://argocd-notifications.readthedocs.io/en/stable/triggers/#default-triggers) as part of the helm chart.

Thank you for this great project.
Bye

Note on DCO:

If the DCO action in the integration test fails, one or more of your commits are not signed off. Please click on the *Details* link next to the DCO action for instructions on how to resolve this.

Checklist:

* [x] I have bumped the chart version according to [versioning](https://github.com/argoproj/argo-helm/blob/master/CONTRIBUTING.md#versioning)
* [x] I have updated the chart changelog with all the changes that come with this pull request according to [changelog](https://github.com/argoproj/argo-helm/blob/master/CONTRIBUTING.md#changelog).
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/tree/master/community#contributing-to-argo).
* [ ] My build is green ([troubleshooting builds](https://argoproj.github.io/argo-cd/developer-guide/ci/)).

Changes are automatically published when merged to `master`. They are not published on branches.

### CODE REVIEW

File: charts/argocd-notifications/templates/configmap.yaml [hunk 1]
```diff
@@ -21,6 +21,12 @@ data:
   {{- with .Values.templates }}
     {{- toYaml . | nindent 2 }}
   {{- end }}
+  {{- with .Values.defaultTriggers }}
+  defaultTriggers: |
+    {{- range . }}
+    - {{ . }}
+    {{- end }}
+  {{- end }}
```
Comment (mkilchhofer) [inline charts/argocd-notifications/templates/configmap.yaml:29]: Please use the same code style as all other settings -> with / toYaml

File: charts/argocd-notifications/Chart.yaml [hunk 2]
```diff
@@ -16,3 +16,4 @@ maintainers:
 annotations:
   artifacthub.io/changes: |
     - "[Added]: Initialize Changelog"
+    - "[Added]: Support of 'defaultTriggers'"
```
Comment (mkilchhofer) [inline charts/argocd-notifications/Chart.yaml:19]: The old changelog entry needs to be removed
```suggestion
    - "[Added]: Support of 'defaultTriggers'"
```

**Review by mkilchhofer — CHANGES_REQUESTED** [review]
Thank you for your contribution. Please have a look at my comments.

**Comment by mkilchhofer** [comment]
Ahh, we don't need this change. `.Values.triggers` is a generic section for triggers. The `defaultTrigger` is just one special trigger which can be defined. Please rework this PR to only include defaultTriggers as a commented out example below `.Values.triggers`

**Review by mkilchhofer — CHANGES_REQUESTED** [review]
Ahh, we don't need this change. `.Values.triggers` is a generic section for triggers. The `defaultTrigger` is just one special trigger which can be defined. Please rework this PR to only include defaultTriggers as a commented out example below `.Values.triggers`

```diff
diff --git a/charts/argocd-notifications/values.yaml b/charts/argocd-notifications/values.yaml
index ff5ca14..8e7c61f 100644
--- a/charts/argocd-notifications/values.yaml
+++ b/charts/argocd-notifications/values.yaml
@@ -346,6 +346,8 @@ triggers:
   #     send:
   #     - app-sync-succeeded
   #     when: app.status.operationState.phase in ['Succeeded']
+  # For more information: https://argocd-notifications.readthedocs.io/en/stable/triggers/#default-triggers
+  # defaultTriggers: |
+  #   - on-sync-status-unknown

 bots:
   # For more information: https://argocd-notifications.readthedocs.io/en/stable/bots/overview/
```

**Comment by shortsn** [comment]
> Ahh, we don't need this change. `.Values.triggers` is a generic section for triggers. The `defaultTrigger` is just one special trigger which can be defined. Please rework this PR to only include defaultTriggers as a commented out example below `.Values.triggers`
> 
> ```diff
> diff --git a/charts/argocd-notifications/values.yaml b/charts/argocd-notifications/values.yaml
> index ff5ca14..8e7c61f 100644
> --- a/charts/argocd-notifications/values.yaml
> +++ b/charts/argocd-notifications/values.yaml
> @@ -346,6 +346,8 @@ triggers:
>    #     send:
>    #     - app-sync-succeeded
>    #     when: app.status.operationState.phase in ['Succeeded']
> +  # For more information: https://argocd-notifications.readthedocs.io/en/stable/triggers/#default-triggers
> +  # defaultTriggers: |
> +  #   - on-sync-status-unknown
> 
>  bots:
>    # For more information: https://argocd-notifications.readthedocs.io/en/stable/bots/overview/
> ```

🤯  i did not realize that this is possible. Thanks for pointing out.

File: charts/argocd-notifications/Chart.yaml [hunk 3]
```diff
@@ -3,7 +3,7 @@ appVersion: 1.1.1
 description: A Helm chart for ArgoCD notifications, an add-on to ArgoCD.
 name: argocd-notifications
 type: application
-version: 1.4.1
+version: 1.5.0
```
Comment (mkilchhofer) [inline charts/argocd-notifications/Chart.yaml:6]: Bump patch version now would be sufficient

File: charts/argocd-notifications/Chart.yaml [hunk 4]
```diff
@@ -15,4 +15,4 @@ maintainers:
   - name: andyfeller
 annotations:
   artifacthub.io/changes: |
-    - "[Added]: Initialize Changelog"
+    - "[Added]: Example for 'defaultTriggers'"
```
Comment (mkilchhofer) [inline charts/argocd-notifications/Chart.yaml:18]: Fix NOEOL error and add an empty line after last line :-)

**Review by mkilchhofer — CHANGES_REQUESTED** [review]

**Review by mkilchhofer — APPROVED** [review]

**Pull request merged**